packages/nextjs-template/package.json

```diff
@@ -21,12 +21,15 @@
     "cypress:open": "cypress open",
     "cypress:run": "cypress run",
     "clean": "yarn rimraf public data out",
-    "rimraf": "npx rimraf"
+    "rimraf": "npx rimraf",
+    "test": "playwright test",
+    "test:skipbuild": "SKIP_BUILD=1 playwright test"
   },
   "dependencies": {
     "@chakra-ui/react": "^1.6.7",
     "@dendronhq/common-all": "^0.80.0",
     "@dendronhq/common-frontend": "^0.80.0",
+    "@playwright/test": "^1.18.1",
```

Comment:
can this be a test dependency instead of a dependency?

Reply:
Sure